web/js/components/map/loading-spinner.js

```diff
@@ -4,21 +4,38 @@ import PropTypes from 'prop-types';
 import { Spinner } from 'reactstrap';
 
 function LoadingIndicator({ msg, isLoading }) {
+  const mediaQuery = window.matchMedia('(max-width: 530px)');
+
   const spinnerStyle = {
     position: 'absolute',
     top: 10,
     left: 300,
     zIndex: 999,
   };
 
+  const mobileSpinnerStyle = {
+    position: 'absolute',
+    top: 10,
+    left: 130,
+    zIndex: 999,
+  };
+
+  if (mediaQuery.matches) {
```

Comment:
Instead of duplicating the return statements, can you do the conditional check where the style is declared? For example:

```const spinnerStyle = mediaQuery.matches ? {  mobile styles } : { default styles };```

or

```<div styles={mediaQuery.matches ? mobileSpinnerStyle : spinnerStyle}>```